website/src/reference/builder/member/getter.md

```diff
@@ -0,0 +1,77 @@
+# `getter` :microscope:
```

Comment:
One more thing. There is a table of all attributes in `website/src/reference/builder.md`, it should be updated